README.md

```diff
@@ -1,13 +1,15 @@
 # Image Cleanup Client
 
+![](https://img.shields.io/docker/pulls/appuio/image-cleanup)
+![](https://img.shields.io/github/v/release/appuio/image-cleanup)
+![](https://img.shields.io/github/license/appuio/image-cleanup)
+
```

Comment:
I'd suggest to align that with how we did it on some of the other container repositories, e.g.

* [container-oc](https://github.com/appuio/container-oc/blob/master/README.md)
* [docker-concierge](https://github.com/vshn/docker-concierge/blob/master/README.md)
* [docker-modulesync](https://github.com/vshn/docker-modulesync/blob/master/README.md)
* [modsecurity-docker](https://github.com/vshn/modsecurity-docker/blob/master/README.md)